index.html

```diff
@@ -124,6 +137,10 @@ <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
     </div>
   </section>
   <section class="about">
+    <div class="images_about">
+    <img src="/img/imgdes/img_about1.png" alt="background_about1" class="img_about1">
+    <img src="/img/imgdes/img_about2.png" alt="background_about2" class="img_about2">
+    </div>
```

Comment:
- [ ] These background images are still not showing up. kindly consider fixing them by the same method I described in the hero section.
your design
![oi](https://user-images.githubusercontent.com/96683843/213874885-e2720470-b3b1-475d-b916-d981ef6442d4.jpg)